### ISSUE TITLE
Can I use PyInstaller hook with fbs?

### ISSUE THREAD

**Issue by a7744hsc** [mherrmann/fbs#75]
When I freeze a project with PySide, there comes a error:
```
4833 ERROR: Can not find path ./libshiboken2.abi3.5.12.dylib (needed by /Users/xxx/PROJECTS/python/testFBS/fbsenv/lib/python3.7/site-packages/PySide2/QtGui.abi3.so)
4847 ERROR: Can not find path ./libshiboken2.abi3.5.12.dylib (needed by /Users/xxx/PROJECTS/python/testFBS/fbsenv/lib/python3.7/site-packages/PySide2/QtNetwork.abi3.so)
```
I can fix this for a standard project with PyInstaller hook, I wonder how can I fix a fbs project with PyIntaller? 

I am not sure if this is an issue, but I have searched around for how to use pyinstaller hook with fbs and nothing found.

**Comment by mherrmann** [comment]
fbs ist based on PyInstaller. Does this help you?

**Comment by mherrmann** [comment]
And are you using vanilla Python, or Anaconda?

**Comment by SidneyGuenther** [comment]
You can edit/add PyInstaller hooks for fbs projects the same way you would do for a "default" project.
Just the location is different.

The hook-directory for your fbsenv should be here:
`/Users/xxx/PROJECTS/python/testFBS/fbsenv/lib/python3.7/site-packages/PyInstaller/hooks`

**Comment by asukakenji** [comment]
The workaround could be found here:
https://github.com/mherrmann/fbs/issues/146#issuecomment-531537851

Basically, I copied `libshiboken2.abi3.xx.yy.dylib` (where `xx` and `yy` are version numbers) from `site-packages/shiboken2/` to both `site-packages/PyInstaller/hooks/` and `site-packages/PySide2/`, then the build was executed successfully.

However, the "*.app" was still not built correctly. When it was executed, the error `LSOpenURLsWithRole() failed with error -10810` occurred. After some researches, I found this comment to be helpful:
https://stackoverflow.com/questions/50580039/creating-pyinstaller-from-my-script-missing-pyside2-qtxml-in-the-result#comment104850986_55771925

Basically, I added the following import to the main source file fix the problem:
```py
from PySide2 import QtXml
```
